decidim-generators/lib/decidim/generators/install_generator.rb

```diff
@@ -73,6 +73,12 @@ def smtp_environment
         end
       end
 
+      def skip_gemfile_if_defined
```

Comment:
This was removed with Rails 7, but we actually use this option for the `test_app`, so I introduced it.